### ISSUE TITLE
type 'Null' is not a subtype of type 'String'

### ISSUE THREAD

**Issue by RasulM97** [IsaiasCuvula/todo#2]
Hello,
I cloned your project for testing and when it started to run, an error showed, how can I solve it?
![image](https://user-images.githubusercontent.com/85100446/179061649-f299ea5b-66b5-4718-bbe4-fc063de4080f.png)

**Comment by IsaiasCuvula** [comment]
Try to check if you are using GetMaterialApp and if you have all dependencies added, and if you are using the last version of Flutter

**Comment by RasulM97** [comment]
The problem is reading from database in setting_controller, because dart sees the null at first run, it supposed to be a string.

**Comment by IsaiasCuvula** [comment]
Yeah I got, I updated the code, the error was thrown because at the first run the username is null (because there is no name in shared preferences) the way to solve this bug, just add a default name , when reading the name on setting_controller file, ![image](https://user-images.githubusercontent.com/68303716/179210448-c2051bed-6037-4e37-b303-3c3f55b226b3.jpeg) 


Update like this and uninstall the app and run again ! Let me know if you're facing another issue please
